orchestration/dagster_orchestration/hca_orchestration/tests/test_resources.py

```diff
@@ -47,7 +47,7 @@ class DataflowBeamRunnerTestCase(unittest.TestCase):
         'DATAFLOW_SUBNET_NAME': 'snubnet',
         'GCLOUD_REGION': 'ec-void1',
         'DATAFLOW_WORKER_MACHINE_TYPE': 'most-expensive-4',
-        'DATAFLOW_STARTING_WORKERS': '2',   # these are marked as ints behind the scenes, but
+        'DATAFLOW_STARTING_WORKERS': '2',  # these are marked as ints behind the scenes, but
```

Comment:
wondering if we have some kind of drift in linter settings here  - this doesn't get enforced on my end, and it was indented that way to align with the second line of the comment below it

Reply:
I must've touched it manually, putting it back and running the linter doesn't result in a modification by the linter. I'll change it back to avoid unnecessary history.

Reply:
Turns out this was intellij being helpful, I'm looking for a way to have it respect our linter settings.